src/__tests__/parser.test.ts

```diff
@@ -368,6 +399,10 @@ describe('parse', () => {
       # description: address book
       # format: map(text,phone_number)
       ADDRESS_BOOK=dom,+12223334444;phil,+13334445555
+
+      # description: config file
+      # format: file(json)
```

Comment:
I think because it's an implementation detail on the CodeExchange side I'd not enforce it for the time being. Maybe we can add a test case to the manifest generation code inside function-templates. I'd be fine with that.

Reply:
I'm holding off on this change for now, since it sounds like the test for this might be implemented in `function-templates` instead.